src/org/sparkboard/server/server.clj

```diff
@@ -400,104 +292,20 @@
             (ring.http/unauthorized))))
       (f req))))
 
-
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 (defn slack-api-handlers []
   (merge
-    @view/registry
-    {:shortcut/main-menu
-     ;; TODO existing shortcuts don't have callback_id set?
-     (fn [context]
-       (slack/web-api "views.open" {:auth/token (:slack/bot-token context)}
-                      {:trigger_id (-> context :slack/payload :trigger_id)
-                       :view (hiccup/->blocks-json (screens/shortcut-modal context))}))
-     :event/app_home_opened
-     (fn [context]
-       (case (-> context :slack/event :tab)
-         "home" (update-user-home-tab! context)
-         nil))
-
-     :event/team_join link-account-for-new-user
-
-     :view_submission/team-broadcast-modal-compose
```

Comment:
Am I correct that all these handlers are now covered by the in-place callbacks in `defview`? In other words most of these haven't moved but rather just go away because they aren't necessary to have in this separate namespace.

Reply:
correct - they are now defined 'in place'. Less spooky action at a distance